addons/resource/views/resource_view.xml

```diff
@@ -76,7 +75,7 @@
             <field name="arch" type="xml">
                 <tree string="Working Time">
                     <field name="name"/>
-                    <field name="manager"/>
+                    <field name="manager_id"/>
                     <field name="company_id" groups="base.group_multi_company"/>
                 </tree>
             </field>
```

Comment:
From [this line]  (https://github.com/odoo-dev/odoo/blob/master-resource-new-api-bba/addons/resource/views/resource_view.xml#L124), remove `oe_inline` class, and use `o_row` if necessary (also check UI with enterprise after applying changes). Same remarks applies to the whole file.